tests/test_integrated_channels/test_sap_success_factors/test_exporters/test_content_metadata.py

```diff
@@ -32,55 +32,101 @@ def setUp(self):
 
     @ddt.data(
         (
+            #advertised course run has price, use it
```

Comment:
to get quality CI check to pass, you need to change these comments to have a space after the hashmark:

`E265 block comment should start with '# '`